webpack.config.js

```diff
@@ -19,6 +19,10 @@ module.exports = {
     plugins.html({
       template: `${__dirname}/examples/index.html`,
     }),
+    plugins.define({
+      __SVG__: false,
+    }),
     plugins.extractText(),
+    plugins.uglify({ parallel: 4 }),
```

Comment:
oops, just a quick test of the DCE

Reply:
I have always wondered why it takes three tools: uglify rollup and GCC for DCE.

Reply:
i guess, GCC probably replaces ugilfy in this case?

Reply:
I guess. And rollup does a bit of elimination too. And I'm not talking about tree shaking of ES6. I have seen

```
if (true) {
// code
}
```

being turned to

```
{
// code 
}
```
by rollup

Reply:
interesting!

Reply:
You don’t need Uglify if you use GCC. In fact adding Uglify can make things worse if you have GCC-d code